### PR TITLE
Update to 2022.07.01-6 and remove rsync module (#9)

### PR DESCRIPTION
**Pull request by Ferenc-**
Signed-off-by: Ferenc Géczi <ferenc.gm@gmail.com>

### CODE REVIEW

File: changelog-2022.07.01-6.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+@@ -107,6 +107,21 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2022.07.01-6" date="2022-07-01">
++     <description>
++       <ul>
++         <li>Improved/fixed the drawing of pattern lines</li>
++         <li>Icons for information=board/map</li>
++	  <li>Fixed categories search with stop words</li>
```
Comment (biodranik) [inline changelog-2022.07.01-6.patch:14]: Indentation is not beautiful.

**Review by biodranik — APPROVED** [review]

File: changelog-2022.07.01-6.patch (hunk 1 above)
Comment (Ferenc-) [reply changelog-2022.07.01-6.patch:14]: Could you be more specific? I don't see any difference compared to the original `app.organicmaps.desktop.metainfo.xml` or the previous patch `changelog-2022.06.18-2.patch`.

Comment (biodranik) [reply changelog-2022.07.01-6.patch:14]: I assume you mixed tabs and spaces.

image

Comment (Ferenc-) [reply changelog-2022.07.01-6.patch]: Ohhh. That should be fixed in the latest commit.
Do we want to handle these kinds of issues automatically?
For example in gitconfig or a GH workflow or something similar?

Comment (Ferenc-) [reply changelog-2022.07.01-6.patch]: Hi @biodranik is there anything else needed before merging this PR?

**Review by biodranik — APPROVED** [review]

**Pull request merged**